logrotate.h

```diff
@@ -45,9 +45,9 @@ struct logInfo {
     char *oldDir;
     enum { ROT_HOURLY, ROT_DAYS, ROT_WEEKLY, ROT_MONTHLY, ROT_YEARLY, ROT_SIZE
             } criterium;
-    unsigned long long threshhold;
-    unsigned long long maxsize;
-    unsigned long long minsize;
+    size_t threshhold;
```

Comment:
This does not look right.  You changed the internal representation and printf conversions to ```size_t``` despite the value is assigned from ```unsigned long long``` and later explicitly converted to ```off_t``` before a comparison.  Looks like just too many conversions for nothing.  Why did you choose ```size_t```?